api/app/caching.py

```diff
@@ -98,13 +98,13 @@ def cache_file(url: str, prefix: str, extension: str = "cache") -> FilePath:
 
 
 @timed
-def cache_geojson(geojson: GeoJSON, prefix: str) -> FilePath:
+def cache_geojson(geojson: GeoJSON, prefix: str, cache_key: str) -> FilePath:
```

Comment:
Please explicit the inputs here, esp. the cache_key and how it should be used.

Reply:
Added some add comments and updated the params to be more explicit